src/plays/github-user-finder/components/InputCard.css

```diff
@@ -0,0 +1,182 @@
+#input {
```

Comment:
One common rule is, let's not override the tag selectors directly and not to use very common id or class name. Otherwise, you will end up conflicting with the other area of the app.

So here is my suggestion: Add a prefix(unique) to all the selectors of this file.

For example, 
```css
.guf #input {


}
.guf button {

}
```
So on..

Here `.guf` a unique name. You can give anything. I took from `GitHub(g) User(u) Finder(f)`.

Now you can use this unique class name `guf` in your component's JSX and modify the line as

```html
<div className="play-details-body guf">
```

This will make sure no override.